addons/hw_drivers/iot_handlers/drivers/PrinterDriver_L.py

```diff
@@ -94,18 +93,26 @@ def supported(cls, device):
         if device.get('supported', False):
             return True
         protocol = ['dnssd', 'lpd', 'socket']
-        if any(x in device['url'] for x in protocol) and device['device-make-and-model'] != 'Unknown' or 'direct' in device['device-class']:
+        if (
+                any(x in device['url'] for x in protocol)
+                and device['device-make-and-model'] != 'Unknown'
+                or (
+                    'direct' in device['device-class']
+                    and 'serial=' in device['url']
```

Comment:
Note to self:
On raspberry pi 5 we need to add the "serial" check otherwise some fake printers triggering errors are added in the list